merlin/exceptions/__init__.py

```diff
@@ -103,3 +103,12 @@ class NoWorkersException(Exception):
 
     def __init__(self, message):
         super().__init__(message)
+
```

Comment:
wouldn't : 
```python
class MerlinInvalidTaskServerError(Exception):
      pass
```


be enoguh?

Reply:
I just tried it an yes it is! I could've sworn the init was needed in the past; maybe it was a python 3.6/3.7 thing